### PR TITLE
riscv: rename sbadaddr -> stval

### PR DESCRIPTION
**Pull request by mbrcknl**
This updates the proofs for https://github.com/seL4/seL4/pull/231.

### CODE REVIEW

File: proof/crefine/RISCV64/VSpace_C.thy [hunk 1]
```diff
@@ -251,7 +251,7 @@ lemma handleVMFault_ccorres:
     prefer 3 apply simp
    apply (simp add: handleVMFault_def handleVMFault'_def liftE_bindE condition_const
                     ucast_ucast_mask bind_assoc)
-   apply (rule corres_split[OF _ read_sbadaddr_ccorres[ac]], drule sym, clarsimp)
+   apply (rule corres_split[OF _ read_stval_ccorres[ac]], drule sym, clarsimp)
```
Comment (Xaphiosis) [inline proof/crefine/RISCV64/VSpace_C.thy:254]: Practicing using the github interface... do you remember what the `[ac]` attribute does?

**Comment by Xaphiosis** [comment]
Looks good, a good practice issue for external github.

File: proof/crefine/RISCV64/VSpace_C.thy (hunk 1 above)
Comment (mbrcknl) [reply proof/crefine/RISCV64/VSpace_C.thy:254]: IIRC, that's a gadget to convert a `ccorres` rule to a `corres` rule via AutoCorres.

**Comment by lsf37** [comment]
I've made up label `seL4-PR` for those l4v PRs that require a simultaneous seL4 PR.

The `update branch` button generates merges from `master` into this `PR`. These disappear on `Rebase and merge`, so if you're happy with the result, it's fine to push the merge button.

**Comment by mbrcknl** [comment]
The AInvs test failure is expected, since these are proof updates for an seL4 change that has been merged into our internal seL4 repository, but hasn't yet been pushed out to the public seL4 GitHub repository.

**Comment by lsf37** [comment]
That's fine. It'll probably take even longer than usual, because it's not just the sync between the repos that has to happen, but also a full release push to the github verification-manifest repo with the specific hashes that bamboo reported. That last step needs all other projects on top of seL4 to also report back green.

**Comment by lsf37** [comment]
> That last step needs all other projects on top of seL4 to also report back green.

Maybe we should change that, actually. One more for the todo list to discuss.

**Pull request merged**